install_config/persistent_storage/persistent_storage_ceph_rbd.adoc

```diff
@@ -34,27 +40,71 @@ storage provider.
 ====
 
 [[ceph-provisioning]]
-
 == Provisioning
+
 Storage must exist in the underlying infrastructure before it can be mounted as
-a volume in OpenShift. To provision Ceph volumes in OpenShift, the
-following are required:
+a volume in OpenShift. To provision Ceph volumes in OpenShift, the following
+information is required:
 
-- Existing Ceph volumes, to be defined in the persistent volume object
-- The `*PersistentVolume*` API
+- The ceph key (in base64) to be used as an OpenShift secret object.
+- The ceph image name.
+- The file system type on top of the block storage (e.g., ext4).
```

Comment:
So far we have not, and looking around in RHEL documentation, it looks like _most_ of the time it is not. So I guess we can leave it plain.